### PR TITLE
Fix versioning for axa-text

### LINKED ISSUE #1979: Text: If version is filled, Variant property does not work
When Version is filled in createAxaText, the Variant property does not Work for Text.

## Expected Behavior
Variant property of Text should work independently of Version

## Steps to Reproduce
1. Create AxaText with version

const version = "esales";
const AxaTextReact = createAXAText(createElement, version);
export const SomeText = () => <AxaTextReact variant="bold">Text</AxaTextReact>

Der text ist nicht bold wie er sein sollte
Wenn mann aber die Version entfernt funktioniert das wie erwartet

## Context (Environment)
We version all components of PL due to possible conflicts with other pods. In this case it is not possible

### PR DESCRIPTION
**Pull request by markus-walther**
Fixes #1979.

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.

### CODE REVIEW

**Review by danyball — APPROVED** [review]
minimal change, nice

File: src/components/10-atoms/text/index.js [hunk 1]
```diff
@@ -16,9 +15,7 @@ class AXAText extends NoShadowDOM {
   }
 
   static get styles() {
-    return css`
-      ${unsafeCSS(styles)}
-    `;
+    return styles;
```
Comment (01231) [inline src/components/10-atoms/text/index.js:18]: shouldn't this be changed on all components then?

Comment (01231) [reply src/components/10-atoms/text/index.js:18]: for example cookie-disclaimer has `${unsafeCSS(styles)}`

Comment (markus-walther) [reply src/components/10-atoms/text/index.js:18]: Yes, it should. But that's another issue, I opened https://github.com/axa-ch/patterns-library/issues/2006.

**Pull request merged**